src/mcts/search.cc

```diff
@@ -319,6 +319,9 @@ void Search::MaybeTriggerStop() {
   if (limits_.time_ms >= 0 && GetTimeSinceStart() >= limits_.time_ms) {
     FireStopInternal();
   }
+  if (limits_.depth >= 0 && cum_depth_ / (total_playouts_ ? total_playouts_ : 1) >= (unsigned int)limits_.depth) {
```

Comment:
According to the style guide:
1. C-style casts are not allowed (use static_cast)
2. Lines longer than 80 characters are not allowed.   It's recommended to run clang-format to format the code as it also does other changes.

I've done it for this snippet though:
```suggestion
if (limits_.depth >= 0 &&
    cum_depth_ / (total_playouts_ ? total_playouts_ : 1) >=
        static_cast<unsigned int>(limits_.depth)) {
}
```

Also add a comment before the `if` statement.

Reply:
Seems you added a '}' too much?